awx/sso/pipeline.py

```diff
@@ -276,7 +277,7 @@ def _check_flag(user, flag, attributes, user_flags_settings):
             # if they don't match make sure that new_flag is false
             else:
                 logger.debug(
-                    "Refusing %s for %s because attr %s (%s) did not match value '%s'"
+                    "For %s on %s attr %s (%s) did not match expected value '%s'"
```

Comment:
For flag on user.username attr...

Is `For` the right word here?

Reply:
I'm opened to suggestions but this would resolve to something like:
`For superuser on awx_unpriv attr username (['awx_unpriv']) did not match expected value 'a'`

I used username as the attribute which is why you get that `['awx_unpriv']`.